flow360/component/simulation/translator/utils.py

```diff
@@ -156,15 +156,42 @@ def _is_last_of_type(lst, obj):
     return lst[last_index] == obj
 
 
+def getattr_recursive(obj, attr_name: (str, list), *args):
```

Comment:
```suggestion
def getattr_by_path(obj, path: Union[str, list], *args):
```

Reply:
![image](https://github.com/user-attachments/assets/016cf9a2-a81c-4d4e-a22c-6dca992bf807)

Reply:
This did not get caught by pylint?

Reply:
No it wasn't caught. Union caused trouble for Python3.9 a few times